src/extension.ts

```diff
@@ -372,45 +376,51 @@ export class Extension {
     if (!project)
       return;
 
+    if (request.continuous) {
+      this._watchSupport.addToWatch(project, request, cancellationToken!);
+      return;
+    }
+
     // VSCode will issue several test run requests (one per enabled run profile). Sometimes
     // these profiles belong to the same config and we only want to run tests once per config.
     // So we collect all requests and sort them out in the microtask.
-    if (!this._projectsScheduledToRun) {
-      this._projectsScheduledToRun = [];
-      this._projectsScheduledToRun.push(project);
+    if (!this._itemsScheduledToRun) {
+      this._itemsScheduledToRun = [];
+      this._itemsScheduledToRun.push({ project, request });
       // Make sure to run tests outside of this function's control flow
       // so that we can create a new TestRunRequest and see its output.
       // TODO: remove once this is fixed in VSCode (1.78?) and we
```

Comment:
if we bump, we can also remove this hack I guess?

Reply:
follow up?